lib/utils.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import { exec } from 'teen_process';
+import { fs } from 'appium-support';
+import log from './logger';
+
+
+async function findAppPath (bundleId) {
+  let stdout;
+  try {
+    const output = await exec('/usr/bin/mdfind', [
+      `kMDItemCFBundleIdentifier=${bundleId}`
+    ]);
+    stdout = output.stdout;
+  } catch (e) {
+    log.info(e.stderr || e.message);
+    return null;
+  }
+
+  const matchedPaths = _.trim(stdout)
+    .split('\n')
+    .map((p) => _.trim(p))
+    .filter(Boolean);
+  const appsMapping = {};
+  for (const matchedPath of matchedPaths) {
+    if (await fs.exists(matchedPath)) {
+      const {mtimeMs} = await fs.stat(matchedPath);
+      appsMapping[matchedPath] = mtimeMs;
+    }
+  }
+  if (_.isEmpty(appsMapping)) {
+    return null;
+  }
+  // Get the most recent app
+  return _.toPairs(appsMapping)
```

Comment:
How about printing available apps as info log?
It should be helpful to see what app paths are available on the user environment

Reply:
add a debug log